crates/aptos/src/account/list.rs

```diff
@@ -4,34 +4,58 @@
 use crate::common::types::{
     CliCommand, CliConfig, CliError, CliTypedResult, ProfileOptions, RestOptions,
 };
-use aptos_rest_client::{types::Resource, Client};
+use aptos_rest_client::Client;
 use aptos_types::account_address::AccountAddress;
 use async_trait::async_trait;
-use clap::Parser;
+use clap::{ArgEnum, Parser};
+use serde_json::json;
+use std::str::FromStr;
 
-/// Command to list resources owned by an address
+#[derive(ArgEnum, Clone, Copy, Debug)]
+enum ListQuery {
+    Modules,
+    Resources,
+}
+
+impl FromStr for ListQuery {
+    type Err = &'static str;
+
+    fn from_str(s: &str) -> Result<Self, Self::Err> {
+        match s.to_lowercase().as_str() {
+            "modules" => Ok(ListQuery::Modules),
+            "resources" => Ok(ListQuery::Resources),
+            _ => Err("Invalid query. Valid values are modules, resources"),
+        }
+    }
+}
+
+/// Command to list resources or modules owned by an address
```

Comment:
Let's replace `resources or modules` with `items` and then we don't have to update the comments in the future

Reply:
Updated.